### DISCUSSION TITLE
How to edit suricata config?

### DISCUSSION THREAD

**Discussion by techstartupexplorer** [Security-Onion-Solutions/securityonion#9962] [General]
Hello,
how to edit suricata config? the documentation doesn't provide much info using salt...

**Comment by robbiemarshall** [comment]
Are you wanting to edit the suricata config itself, or one or more of the rules?

**Reply by techstartupexplorer** [reply]
yes i want to edit suricata config but its on read-only whenever i tgry to edit it via docker exec suricata /bin/bash

**Reply by robbiemarshall** [reply]
You'll want to edit the suricata.yaml file.  Documentation on how to do this can be found here: https://docs.securityonion.net/en/2.3/suricata.html#configuration
